topology/pre-process-object.c

```diff
@@ -1051,7 +1069,8 @@ static const struct build_function_map *tplg_object_get_map(struct tplg_pre_proc
 			return &object_build_map[i];
 
 		if (!strcmp(class_type, "Dai") &&
-		    !strcmp(object_build_map[i].class_type, "Dai"))
+		    !strcmp(object_build_map[i].class_type, "Dai")  &&
+			!strcmp(object_build_map[i].class_name, ""))
```

Comment:
is this change really required?

Reply:
actually not.... not sure why I added it originally. Will remove